daemon/emer-daemon.c

```diff
@@ -1005,18 +933,8 @@ handle_login_manager_signal (GDBusProxy *dbus_proxy,
 {
   if (g_strcmp0 ("PrepareForShutdown", signal_name) == 0)
     {
-      gboolean shutting_down;
-      g_variant_get_child (parameters, 0, "b", &shutting_down);
-      if (shutting_down)
-        {
-          flush_to_persistent_cache (self);
-          update_timestamps (self);
-          release_shutdown_inhibitor (self);
-        }
-      else
-        {
-          inhibit_shutdown (self);
-        }
+      flush_to_persistent_cache (self);
```

Comment:
I think the chance a significant amount of events was recorded between those signals is very small - so small that it's not worth complicating the code.
In any case I would not oppose keeping the logic that only does this when `shutting_down` is TRUE